### DISCUSSION TITLE
Request Access to Beta testing

### DISCUSSION THREAD

**Discussion by OlafenwaMoses** [Gen-XR/TheiaEngine#2] [General]
TheiaEngine is currently in beta testing and only available to those who sign up. To get the API url and API key, do the following
- Fill this [form](https://forms.gle/3UrRn3QbudUrJZgS7)
- Comment on this discussion signifying interest and confirm you have filled the form.

**Comment by Amazingct** [comment]
This is beautiful, i am intrested in this

**Reply by OlafenwaMoses** [reply]
@Amazingct Thanks for signing up for the beta testing. You have been granted access and API credentials have been sent to your email.

**Comment by abduIbasit** [comment]
Hi Moses! I signed up for the beta testing, I’m highly interested in exploring the functionality. This is very good work 👍🏾

**Comment by VannyYuan** [comment]
I just fill the form and I hope I can use this api to sort my pictures!

**Comment by WBDC** [comment]
Hi,

I have successfully filled out the form and am excited about the possibility of using TheiaEngine. I'm particularly interested in scene mapping and describing a scene. Looking forward to being a part of the beta testing.

**Comment by OlafenwaMoses** [comment]
@abduIbasit , @WBDC , @VannyYuan  You all have been sent access credentials to the API. Test and drop comments/feedback by creating a new discussion [here](https://github.com/Gen-XR/TheiaEngine/discussions)

**Comment by justsvenja** [comment]
This looks usefull for my Projet im working on.
Gamedevelopment in CryEngine, use intended for Game Cover and Ingame Pictures such as Paintings and Loading Screens.
Credits will be given in the Game no matter what.
Filled the Form too

**Comment by ivedantmistry** [comment]
Hello, I want to try this library for my project "image recognition for medicinal plants using machine learning"

**Reply by ivedantmistry** [reply]
@OlafenwaMoses

**Comment by Plumillon** [comment]
Awesome, I want to be part of it :)
Building with this Theia would be so good!

**Comment by CodyMoltar** [comment]
Looks amazing!
I've filled out the form and would like to participate!

**Comment by one-backstage** [comment]
Добрый день! Очень хочу попробовать)

**Comment by djengi-clan** [comment]
I'm a QA Engineer and I'm just learning Python, would like to experiment with this thanks!

**Comment by crystalties** [comment]
Thanks to Moses, the feature is great and I hope it lets me discover the new continent.

**Comment by 904409516** [comment]
Thanks to Moses.

**Comment by isuixiang** [comment]
Great. I'm looking forward to it!

**Comment by davidezari** [comment]
I have filled the form due I'm searching for an on premise image and video custom object detection solution for one of our customers

**Comment by Klauftoofs** [comment]
Hi,

We are keen to try your product!

**Comment by T0NN0T3K** [comment]
thanks for sharing amazing content,
gonna test this very soon for educational purposes!

**Comment by glenn72945** [comment]
Amazing work! would like to sign up for the beta please, have filled in the form

**Comment by UberHelmsman** [comment]
hi! i have filled the form, i really want to try it out

**Comment by ChikangaTakudzwa** [comment]
Hi Moses, i have filled the form and cant wait to get started and build stuff

**Comment by joakimvivas** [comment]
Hi Moses! It seems amazing your project, I have filled out the form and it would be excellent to have your answer and start playing with the API. Many thanks again!

**Comment by fivesecsofmartaa** [comment]
Hi! I've filled the form and I can't wait to play with the API.

**Comment by sagarneo11** [comment]
Filling the form needs AI Connected, I am way lazy to type it off 😜😜😜😜

**Comment by echai0523** [comment]
你好！我已经填写了表格，我迫不及待地想使用 API。

**Comment by StarTiger** [comment]
Hello. to the devs, thank you for letting us beta for you, I'm in need of fast desktop image processing, for a realtime ai computer control and task completion app. Please consider me for the betatest. Thanks
Anthony Walker

**Comment by 332677437** [comment]
Hello! I've filled out the form and I can't wait to use the API.

**Comment by me-gauravaggarwal** [comment]
Filled the form..looking forward to trying this amazing API.

**Comment by amxv** [comment]
requested access, hope to hear from you soon :)

**Comment by matthiaswernli** [comment]
Hi,

I have successfully filled out the form and am excited about the possibility of using TheiaEngine.

**Comment by giltee** [comment]
I'm very interested in testing the semantic scene mapping, is it available yet or still coming soon?

**Comment by geezercodes** [comment]
Very interesting @OlafenwaMoses . I have filled in the interest form as I am interested.

**Comment by Guoxiang1** [comment]
I really want to experience and try this API, which is very helpful for my paper.I have experienced ImageAI before, and it was a very interesting project.I have filled in the interest form as I am interested.

**Comment by TheHugoHypothesis** [comment]
Hello! filled the form. 
@OlafenwaMoses 

I am a student at the University of São Paulo, who, in collaboration with a PhD professor, will write an article relating elements that generate heat islands in the urban environment of the city of São Paulo. One of the methods used is to use an object recognition AI, in order to recognize elements that form these islands, such as buildings, cars, etc. This AI would be very useful, and if it works well for our use, it will certainly be mentioned in the scientific article by the university.

**Comment by Dartwister72** [comment]
Hello there I am signing up for the beta testing of the model. I want to see its potential result by organizing my bulky library, and I am highly interested in testing the model.

**Comment by shashank-2102** [comment]
Hi, I have filled in the form.

I plan to use the model for software log analysis as part of my Bachelor end project!

**Comment by wriwen** [comment]
Hello, I have filled out the form and explained my ideas. I hope to get the opportunity to test and realize my ideas. Thank you for your efforts.

**Comment by cozmo14047** [comment]
> TheiaEngine is currently in beta testing and only available to those who sign up. To get the API url and API key, do the following
> 
> * Fill this [form](https://forms.gle/3UrRn3QbudUrJZgS7)
> * Comment on this discussion signifying interest and confirm you have filled the form.

Hi there, we have filled the form and are wanting to use this in robotics. many thanks

**Comment by dav2010ID** [comment]
Looks amazing!
I've filled out the form and would like to participate!

**Comment by bliu221** [comment]
I am excited to try the beta! I have filled out the form and would like to participate.

**Comment by mihirhirave** [comment]
This is wonderful! Just filled out the form. I'm a student and can't wait to try it out.

**Comment by muhammad-imaz** [comment]
This looks awesome, would love to try it ASAP.

**Comment by Sadraiii** [comment]
I just filled out the form and as a student I can't wait to try this out, and hopefully include it in my thesis work.

**Comment by cozmo14047** [comment]
@OlafenwaMoses is still going ?

**Comment by FallingMapleSJ** [comment]
I would really love it if I could join! Thanks!

**Comment by olawalejuwonm** [comment]
I can't wait to test this out! I just signed up for beta testing

**Comment by darshan2101** [comment]
I'd love to test it out. would be great use in my projects

**Comment by SvetlanaVor** [comment]
I have filled out the form. I am keen on testing it!

**Comment by NGalletftv** [comment]
Hi, 
Not sure that it's always possible but I fill in the form and I would be happy to test your amazing tool. 
Thank you

**Comment by scarfon** [comment]
Hi @OlafenwaMoses, I'm thrilled to try the tool, if possible I would love to run it locally on my project for my classes

**Comment by C-zu** [comment]
I have filled the form. I'm looking forward to using it and seeing how it helps my project!

**Comment by Dev-YashRajPurohit** [comment]
I have filled the form, I am currently in my learning phase and just completed python and my really interested in this domain of programming.

**Comment by fahim6285** [comment]
Hi, I have Filled the Form and I would like to participate in this Program as am Interested in Research work on Road Density and Counting the Vehicles.

**Comment by annysource** [comment]
> TheiaEngine is currently in beta testing and only available to those who sign up. To get the API url and API key, do the following
> 
> * Fill this [form](https://forms.gle/3UrRn3QbudUrJZgS7)
> * Comment on this discussion signifying interest and confirm you have filled the form.

Hi, I'm interested! Awesome

**Reply by annysource** [reply]
I have filled the form from other email @OlafenwaMoses

**Comment by DeelanLasrado** [comment]
Hi @OlafenwaMoses! I signed up for the beta testing, I’m highly interested in exploring the functionalities

**Comment by scottiepowell** [comment]
Hi @OlafenwaMoses  I've signed up for the beta testing, i'm interested in exploring this technology for a mobile application

**Comment by 0469546** [comment]
Hi @OlafenwaMoses I've signed up for the beta testing, and I don't receive information about API and URL

**Comment by sai1231** [comment]
Hi @OlafenwaMoses I'm interested and  signed up for the beta testing

**Comment by 0469546** [comment]
I am interested in beta testing, I expect api and url data

пн, 28 апр. 2025 г., 08:19 sai1231 ***@***.***>:

**Comment by Clay-hkf** [comment]
Hello, I am interested in taking the test and I have completed the form, I expect api and url data
